src/docs/demos/modal/modal-demo.controller.js

```diff
@@ -1,13 +1,9 @@
 class controller {
-  constructor ($ngbsModal) {
-    this.$ngbsModal = $ngbsModal;
+  constructor($log, $ngbsModal) {
+    Object.assign(this, { $log, $ngbsModal });
```

Comment:
must admit i never had any experience with it...

And not using uglifyingt yet (yet...)